Unwrap/Activities/Learn/LearnCoordinator.swift

```diff
@@ -55,6 +55,13 @@ class LearnCoordinator: Coordinator, Awarding, Skippable, AlertHandling, AnswerH
         guard let videoURL = Bundle.main.url(forResource: activeStudyReview.title.bundleName, withExtension: "mp4") else { return }
         let player = AVPlayer(url: videoURL)
 
+        // Plays sound when device is physically muted or silent
+        do {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback)
+        } catch {
+            //Didn't work
```

Comment:
Given that we don't actually do anything with the catch block, could this be called just using `try?` and drop the do/catch blocks entirely?